nf_core/pipeline-template/{{cookiecutter.pipeline_slug}}/docs/installation.md

```diff
@@ -1,42 +1,48 @@
-# {{ cookiecutter.pipeline_name }} Installation
+# nf-core/{{ cookiecutter.pipeline_slug }} installation
 
-To start using the {{ cookiecutter.pipeline_name }} pipeline, there are three steps described below:
+To start using the nf-core/{{ cookiecutter.pipeline_slug }} pipeline, follow the steps below:
 
-1. [Install Nextflow](#install-nextflow)
-2. [Install the pipeline](#install-the-pipeline)
-3. Configure the pipeline
-    * [Local installation](configuration/local.md)
-    * [Adding your own system](configuration/adding_your_own.md)
+1. [Install Nextflow](#1-install-nextflow)
+2. [Install the pipeline](#2-install-the-pipeline)
+    * [Automatic](#21-automatic)
+    * [Offline](#22-offline)
+    * [Development](#23-development)
+3. [Pipeline configuration](#3-pipeline-configuration)
+    * [Software deps: Docker and Singularity](#31-software-deps-docker-and-singularity)
+    * [Software deps: Bioconda](#32-software-deps-bioconda)
+    * [Configuration profiles](#33-configuration-profiles)
+4. [Reference genomes](#4-reference-genomes)
+5. [Appendices](#appendices)
+    * [Running on UPPMAX](#running-on-uppmax)
 
 ## 1) Install NextFlow
 Nextflow runs on most POSIX systems (Linux, Mac OSX etc). It can be installed by running the following commands:
 
 ```bash
-# Make sure that Java v7+ is installed:
+# Make sure that Java v8+ is installed:
 java -version
 
 # Install Nextflow
 curl -fsSL get.nextflow.io | bash
 
 # Add Nextflow binary to your PATH:
-mv nextflow ~/bin/
+mv nextflow ~/bin
```

Comment:
I don't really understand this change? `~/bin/` should be fine I think.

Reply:
probably because I just copied the text from the methylseq doc ^^